client/src/main/java/gui/controllers/PlacementSceneController.java

```diff
@@ -61,9 +60,13 @@ public void initialize() {
 
     @FXML
     private void placeRandom() {
+        appInstance.getServerConnection().sendMessage("ASK_FOR_FLEET");
+
+        String message = appInstance.getServerConnection().getMessage();
+
         printingBoard.getChildren().removeIf(node -> node instanceof Shape);
-        FleetDAO fleetDAO = new RandomFleet();
-        fleet = fleetDAO.getGUIFleet();
+        RandomFleet generator = new RandomFleet();
+        fleet = generator.getGUIFleet(message);
         ShipPrinter.placeShips(fleet, printingBoard);
```

Comment:
ShipPrinter has method placeShips? Shouldn't it just be able to print ships?

Reply:
Changed to printFleet.